External/FEXCore/Source/Utils/Allocator.cpp

```diff
@@ -211,7 +215,7 @@ namespace FEXCore::Allocator {
 
           auto MapBegin = std::max(RegionEnd, Begin);
           auto MapEnd = std::min(RegionBegin, End);
-          
+
```

Comment:
This seems better placed here:
```suggestion
          // Store the previous map location
          PreviousMapEnd = MapEnd;
```

Alternatively, MapEnd should be made `const` to make it easier to see you're just storing the unmodified value later on.

Reply:
Moved and made const.